modules/parameter/errors.go

```diff
@@ -0,0 +1,14 @@
+package parameter
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	DefaultCodespace                    sdk.CodespaceType = 5
```

Comment:
I think the DefaultCodespace isn‘t 5. 5 stands for gov. You should use another number.